### DISCUSSION TITLE
不兼容TVBox

### DISCUSSION THREAD

**Discussion by yansong1221** [Hiram-Wong/ZyPlayer#60] [Q&A]
那么多现成的资源为啥丢了呢

**Comment by Hiram-Wong** [comment]
做如下统一解答
1. 首先 tvbox 基于安卓开发，上面有很多jar依赖，这些依赖具有安卓特性，无法在 pc 上运行。（即使你使用pc 的 java 重写一遍也无法兼容安卓）
2. 其次 zy 目前不仅是单纯支持 cms 接口，已经探索了 app 接口以及 drpy 接口
